### DISCUSSION TITLE
Cloudinit template

### DISCUSSION THREAD

**Discussion by danielo515** [tteck/Proxmox#1891] [Ideas]
Hello.
Thank you for this awesome collection of nice scripts.
I was wondering if it would be possible to either add an option to add your github account ssh public key, so you can login to the container, or, even better, use a cloud-init template/configuration that you have on the host and let it configure the server basics.

I want to enable ssh access on all my LXC containers. I usually use one ansible playbook that adds my user, grants it sudo, removes root login etc, but I can't do this because I need to manually edit each LXC before, which is a bit unconvenient.

Thanks in advance

**Comment by yodaphone** [comment]
@tteck this will be very useful, if you can do this
